dojo/tools/tenable/csv_format.py

```diff
@@ -123,9 +156,12 @@ def get_findings(self, filename: str, test: Test):
                     test=test,
                     description=description,
                     severity=severity,
+                    cwe=cwe,
```

Comment:
```suggestion
```

Reply:
I think 0 is the default in models.py. But this line was a left over anyway to be commented out.